src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+interface Props {
+  todo: Todo;
+  onClose: () => void,
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, onClose }) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const {
+    title,
+    completed,
+    id,
+    userId,
+  } = todo;
+
+  useEffect(() => {
+    const getUserFromServer = async () => {
+      try {
+        const userFromServer = await getUser(userId);
+
+        setUser(userFromServer);
+      } catch (error) {
+        setUser(null);
```

Comment:
You need to show the user if there was an error while loading data from the server.